kairon/shared/pyscript/shared_pyscript_utils.py

```
@@ -139,4 +139,14 @@ def delete_schedule_job(event_id: Text, bot: Text):
         if not http_response.get("success"):
             raise AppException(http_response)
         else:
-            logger.info(http_response)
\ No newline at end of file
+            logger.info(http_response)
+
+    @staticmethod
+    def typing_indicator(message_id: str, api_key: str, bot: str):
```

Ruff (0.14.5)

Unused static method argument: `bot`

(ARG004)